packages/mobile/src/app/saga.ts

```diff
@@ -161,6 +161,8 @@ export function* handleDeepLink(action: OpenDeepLink) {
         cicoSuccessParam?.toLowerCase().includes(name.toLowerCase())
       )[0]
       navigate(Screens.CashInSuccess, { provider })
+    } else if (rawParams.path.startsWith('/cash-in-failure')) {
```

Comment:
I think it's prefered if we compare for exact match, like `rawParams.pathname === '/cash-in-failure'`. Not sure if it's a big issue but I've seen @jeanregisser mention this a few times.

Reply:
We can't check for strict equality because sometime providers append information to the redirect link. I've added a comment to explain that

Reply:
We can't check for strict equality because sometimes providers append information to the redirect path. I've added a comment so that it's clear why we are doing it this way